connectivity/check/check.go

```diff
@@ -37,6 +37,7 @@ type Parameters struct {
 	SkipIPCacheCheck      bool
 	Perf                  bool
 	PerfDuration          time.Duration
+	PerfCRR               bool
```

Comment:
If tweaks for other workloads are going to be required, it might be worth using a `string` to specify the mode (e.g. `--perf=crr`) rather than one `bool` for each mode. This is a bit more extensible, and ensures that only one workload type can be specified.

Reply:
Moving CRR to an island wrt the benchmarks should avoid us from having to partition the workload(s) any further. However, it might make sense in a follow on to allow the user to pass a string as you mentioned above in a different PR.

Thoughts?